docs/wallets/helium-wallet-app.mdx

```diff
@@ -0,0 +1,211 @@
+---
+id: helium-wallet-app
+sidebar_label: Helium Wallet App
+slug: /wallets/helium-wallet-app
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Helium Wallet App
+
+<img
+  className="docsheader"
+  src={useBaseUrl("img/wallets/helium-wallet-app/header.png")}
+/>
+
+While you can import wallets that contain Hotspots and Validators, the Helium Wallet
```

Comment:
Feels abrupt to introduce the app with a "but you can't do this".

Suggest something like 
"The Helium Wallet app is the new trusted way to manage your tokens and identity on the Helium Blockchain. This new Wallet App is not meant to manage Hotspots or Validators."